Eastern-Research-Group/mywaterway
# Feature/fix prettier settings

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3342760

## Main Changes:
* Allows arrow function parenthesis in prettier config and applies it across the app

## Steps To Test:
1. Check prettier.config.js files and verify that saving files does not cause prettier to modify the file.
Good file to test saving: client\src\components\pages\Community\components\tabs\IdentifiedIssues.js
